### DISCUSSION TITLE
Parameter Settings

### DISCUSSION THREAD

**Discussion by elcotek** [oobabooga/text-generation-webui#1052] [General]
Is there a easy way to save Paramer settings ?

Like: `Chat parameters, and Custom generation parameters`

Thanks for answering my Question

**Comment by jllllll** [comment]
There are presets in `\text-generation-webui\presets` and some other chat related stuff in `\text-generation-webui\characters`

You should be able to add your own stuff in those folder to select in the webui.

**Reply by elcotek** [reply]
Oki thank You !

**Comment by Sqrlly** [comment]
I do see a presets/Default.txt that includes some of the parameters I want to save.  But the values there are not what it's using when it starts up.  So it doesn't seem like that's actually being used, or something is overriding it.

**Reply by Sqrlly** [reply]
Okay I figured it out.  On the Parameters tab there's a "Generation parameters preset" drop-down that was set to a different one.  You have to select the one you want.  So I made a new one in that folder and can select it now (rather than edit the Default one).

**Comment by RandomLegend** [comment]
How can i see what each parameter is called for the .txt file?

I want to permanently enable "Stop generating at new line character" but i have no clue how it would be called in the .txt file.

**Comment by mattehicks** [comment]
Bumping this.  Need a list of parameter names and values, please.
(AND a way to automatically save the params that are set manually in the WebUI)

**Comment by Braunfeltd** [comment]
I have been trying to figure out all the files for the same either to load a defualt characterc etc.. I see the cli arguments to send but they dont seem to pass.
